app/javascript/mastodon/components/status_content.js

```diff
@@ -46,10 +51,15 @@ export default class StatusContent extends React.PureComponent {
     }
   }
 
+  componentDidMount () {
+    this._updateStatusLinks();
+  }
+
   componentDidUpdate () {
     if (this.props.onHeightUpdate) {
       this.props.onHeightUpdate();
     }
+    this._updateStatusLinks();
```

Comment:
Ah, that makes sense, though it wouldn't be a actual problem for now; because we use `PermaLink` component for links in the placeholder, so it works correctly without `addEventListener`.

However, it makes